src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -1995,15 +1911,9 @@ public void testExpertFilterTwoWindingsTransformerWithInAndNotInOperator() throw
 
     @Test
     public void lineFilterIsEmpty() {
-        HvdcLineFilter hvdcFilter = new HvdcLineFilter(
-                null,
-                null,
-                null,
-                null,
-                new TreeSet<>(),
-                new TreeSet<>(),
-                new NumericalFilter(RangeType.RANGE, 50., null)
-        );
+        HvdcLineFilter hvdcFilter = HvdcLineFilter.builder()
+            .nominalVoltage(new NumericalFilter(RangeType.RANGE, 50., null))
+            .build();
```

Comment:
I think you get `null` instead of `TreeSet<>` in the object as there is not default value in the DTO.

Reply:
hum ... I don't found where I saw this treeset ...